.github/workflows/publish-release.yml

```
@@ -25,3 +25,8 @@ jobs:
       - uses: MetaMask/action-publish-release@v0.1.0
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
```

To allow this, I think this Action and possibly its author have to be added to an org- or repo-wide allow list. Assuming we do that, we should probably just a commit hash as opposed to tag. While the author seems trustworthy, GitHub still advises that you shouldn't use tags to reference Action versions unless you trust the author, and I don't think we **trust** the author.

This hash refers to the latest release, which is more recent than `v3`, but within the same major version: https://github.com/peaceiris/actions-gh-pages/releases/tag/v3.8.0
```suggestion
        uses: peaceiris/actions-gh-pages@068dc23
```

---

We could also consider forking it, so we don't have to adjust permissions and we don't have any risk of it changing out from under us.

---

I've made a new PR #23 that just uses git to deploy instead of a 3rd party action.